env/tensorflow-15/Dockerfile.gpu

```diff
@@ -0,0 +1,9 @@
+FROM linkernetworks/minimal-notebook:master-gpu
+
+LABEL maintainer="Narumi"
+
+# Install Tensorflow
+RUN pip install -U \
+    tensorflow==1.5.1 \
```

Comment:
should be `pip install -U tensorflow-gpu==1.5.1`
but 1.5.1 needs CUDA 9 and cuDNN 7 (CUDA 8 in GPU notebook)